content/blog/2021/08/2021-08-02-cloudevents-plugin-phase-I.adoc

```diff
@@ -0,0 +1,174 @@
+
+CloudEvents specification is enabling that loosely-coupled, event-driven communication between services by enforcing a common language which defines how an event should be emitted and transferred between systems. 
+
+
+== CloudEvents and Jenkins
+
+image:https://cncf-branding.netlify.app/img/projects/cloudevents/stacked/color/cloudevents-stacked-color.png[CloudEvents Specification, link=https://cloudevents.io/, align="center"]
+
+  A specification for describing event data in a common way
+
+____
+* Consistency
+    - Consistent across tools and services. 
+* Accessibility
+    - Common event format means common libraries, tooling, and infrastructure for delivering event data across environments can be used to develop with CloudEvents. 
+* Portability
+    - Easily port event-data across tools, truly leveraging event-driven architecture.
+____
+
+The CloudEvents plugin for Jenkins is developed as an effort to make interoperability between Jenkins and CI/CD tools much easier. The CloudEvents plugin for Jenkins is a GSoC project, and with the help from an amazing team of mentors, this project is aimed at enhancing event-driven interoperability between cloud-native CI/CD tools, making it easier for developers to include Jenkins in their CI/CD pipelines.
+
+With this plugin, Jenkins can send and receive CloudEvents-compliant events to and from a wide variety of CI/CD tools using CloudEvents as their event format. This plugin makes chaining Jenkins with multiple tools like Tekton, Keptn, Knative and more, very easy. 
+
+
+== GSoC Phase 1 - CloudEvents Plugin
+
+=== Using CloudEvents plugin for Jenkins
+
+This plugin allows Jenkins to be configured as a source and sink, which can emit and consume CloudEvents from a range of tools simultaneously. 
+
+==== Jenkins as a Source
+Configuring Jenkins as a Source enables Jenkins to send CloudEvents to a CloudEvents sink. For Phase-I of this project, there is support for HTTP Sinks, however CloudEvents supports various link:https://github.com/cloudevents/spec#cloudevents-documents[protocol bindings]. Moving forward, there will also be support for other protocol bindings supported by CloudEvents. 
+
+
+To use Jenkins as a Source, the following configuration is needed:
+
+1. Click on *Manage Jenkins* in the Root-Actions menu on the left.
+
+image:/images/post-images/2021-08-cloudevents-plugin/manage_jenkins.png[Manage Jenkins]
+
+2. Inside the Manage Jenkins UI, search for *Configure System* under *System Configuration*.
+
```

Comment:
Same here: remove the empty line. When you do a numbered list in asciidoc, the empty line separates the list items and resets the list item counter. So after the empty line, asciidoc expects a new list, not a list starting with `2.`, hence the error message "expected 1, got N".

Reply:
I'm guessing the same goes for the line break between the item and the corresponding image? However, I am not sure if that is going to display correctly, so I have moved the images to the bottom of the list.

Reply:
You can preview your blog post by running `make run` at the top of the jenkins.io git repository. That is what I do :-)